catalog.html

```diff
@@ -6,26 +6,30 @@
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <header class="catalog">
+    <header class="index-header">
       <div class="container">
-        <a href="#" class="logo-index">
-          <img src="http://placehold.it/150x100" alt="">
-        </a>
-        <nav>
-          <ul class="index-menu">
-            <li><a href="#">СТУДИЯ</a></li>
-            <li><a href="#">КЛИЕНТЫ</a></li>
-            <li><span>МАГАЗИН</span></li>
-            <li><a href="#">КОНТАКТЫ</a></li>
-          </ul>
-        </nav>
-        <div class="user-block">
-          <a class="cart" href="#">КОРЗИНА</a>
+        <div class="upper clearfix">
+          <div class="logo">
+            <img  class="logo-index" src="img/nerds-logo.png" alt="">
+          </div>
+          <nav class="menu">
+            <ul class="index-menu">
+              <li><a href="#">СТУДИЯ</a></li>
+              <li><a href="#">КЛИЕНТЫ</a></li>
+              <li><a href="#">МАГАЗИН</a></li>
+              <li><a href="#">КОНТАКТЫ</a></li>
+            </ul>
+          </nav>
+          <div class="cart-block">
+            <a class="login" href="#">КОРЗИНА</a>
+          </div>
+        </div>
+        <div class="title-catalog">
```

Comment:
No need for a container here. You can give the class "title-catalog" to the h1 itself. The fewer elements — the better.